src/storage/mvcc/lock.rs

```diff
@@ -268,8 +293,10 @@ mod tests {
             Some(b"short_value".to_vec()),
             0,
             0,
+            0,
         );
         let v = lock.to_bytes();
         assert!(Lock::parse(&v[..4]).is_err());
+        // TODO: Test min_commit_ts
```

Comment:
Test what?

Reply:
Sorry I forgot to remove this comment. When I starts to write the code I at first didn't check min_commit_ts in this test. so I left a comment here.